src/services/Feed.php

```diff
@@ -16,7 +16,7 @@ public function __construct()
         $this->settings = CalendarFeed::getInstance()->settings;
     }
 
-    public function getFeed()
+    public function getFeed() : ?String
```

Comment:
Seems like this will always return a string

```suggestion
    public function getFeed() : string
```

Reply:
Yeah you're right. 👍